scripts/include.sh/build-dep.sh

```diff
@@ -191,33 +171,9 @@ build_git_osx()
   mkdir -p "$srcdir"
 
   pushd . >/dev/null
-  mkdir -p "$builddir/downloads"
```

Comment:
I'd like to avoid this change or maybe could you add a parameter to that bash function to tweak the the behavior without breaking the rest?